testcases.py

```diff
@@ -1186,7 +1186,12 @@ def check(self) -> TestResult:
 
         last = None
         for p in tr_server:
-            cur = (getattr(p["ip"], "dst"), int(getattr(p["udp"], "dstport")))
+            cur = (
+                getattr(p["ipv6"], "dst")
+                if "IPV6" in str(p.layers)
+                else getattr(p["ip"], "dst"),
```

Comment:
Put these on the same line if possible. Just more readable IMO.

Reply:
This is what `black` auto-formats it to, and if I change it, CI fails.